conda-reqs/chipyard.yaml

```diff
@@ -31,38 +31,44 @@ dependencies:
     - dromajo # from ucb-bar channel - https://github.com/riscv-boom/dromajo
     - firtool==1.30.0 # from ucb-bar channel - https://github.com/ucb-bar/firtool-feedstock
 
+    # misc
+    - autoconf
+    - pre-commit
+    - readline
+    - coreutils
+    - jq
+    - pip
+
     # firemarshal deps
+    - qemu # from ucb-bar channel - https://github.com/ucb-bar/qemu-feedstock
+    - rsync
+    - psutil
+    - doit>=0.34.0
+    - gitpython
+    - humanfriendly
+    - e2fsprogs
```

Comment:
Should we perhaps merge https://github.com/ucb-bar/chipyard/pull/1376 prior to this or combine them in a single PR?

Reply:
`e2fsprogs` will be removed when the Linux bump happens so lets hold off on that for now.